README.md

```diff
@@ -31,7 +31,9 @@ soon as possible.)
 * Tests: **Coming soon.** We rely on three test suites: devcrt, tr1, and [libcxx][]. We need to replace our current test
 harness, which extensively uses Microsoft-internal machinery.
 
-* Continuous Integration: **Coming soon.** We need tests first.
+* Continuous Integration: **In progress.** We have set up Azure Pipelines to validate changes to the repository.

```

Comment:
Nitpick: "We've" would be more consistent with the tone elsewhere. Similarly, "haven't yet hardened it".